pkg/kubelet/envvars/envvars_test.go

```diff
@@ -156,13 +190,6 @@ func TestFromServices(t *testing.T) {
 		{Name: "SCTP_1_PORT_777_SCTP_PORT", Value: "777"},
 		{Name: "SCTP_1_PORT_777_SCTP_ADDR", Value: "1.2.3.4"},
 	}
-	if len(vars) != len(expected) {
-		t.Errorf("Expected %d env vars, got: %+v", len(expected), vars)
-		return
-	}
-	for i := range expected {
-		if !reflect.DeepEqual(vars[i], expected[i]) {
-			t.Errorf("expected %#v, got %#v", vars[i], expected[i])
-		}
-	}
+
+	assert.Equal(t, expected, vars, "env entries")
```

Comment:
Could you share more about the usage of `assertify`? Not opposed to it, but wondering if we should consider it as part of a separate diff?

Reply:
The output on failure is more descriptive and it actually shows where slices differ (the output is very similar to doing `git diff`).

Reply:
The showing where the slices differ makes sense as a value prop to me :)